src/test/mappingHandlers.test.ts

```diff
@@ -1,29 +1,263 @@
 /** @file test against agoric-3-proposals:latest */
 import { subqlTest } from '@subql/testing';
-import { StateChangeEvent, Wallet } from '../types';
+import {
+  BoardAux,
+  OraclePriceDaily,
+  PsmMetricsDaily,
+  ReserveAllocationMetrics,
+  ReserveAllocationMetricsDaily,
+  ReserveMetrics,
+  StateChangeEvent,
+  Vault,
+  VaultManagerGovernance,
+  VaultManagerMetrics,
+  VaultManagerMetricsDaily,
+  VaultStatesDaily,
+  Wallet,
+} from '../types';
 
 // FIXME observed in A3P:latest but not passing tests
 subqlTest(
-  'Wallet and StateChangeEvent in state_change at block 627',
-  627, // block height to process
-  [], // dependent entities
+  'StateChangeEvent in state_change at block 627',
+  627,
+  [],
   [
     new Wallet(
-      'published.wallet.agoric1ee9hr0jyrxhy999y755mp862ljgycmwyp4pl7q',
+      'published.wallet.agoric1rwwley550k9mmk6uq6mm6z4udrg8kyuyvfszjk.current',
```

Comment:
looking at `.current` is more comprehensive but a path without `.current` should also work.